apistructs/approve.go

```diff
@@ -79,7 +79,7 @@ type ApproveDeleteResponse struct {
 // ApproveDetailResponse GET /api/approves/{approveId} 审批详情响应结构
 type ApproveDetailResponse struct {
 	Header
-	ApproveDTO `json:"data"`
+	Data ApproveDTO `json:"data"`
```

Comment:
i dont know what purpose to changes this and will it break api compatibility?

Reply:
if I keep the struct, the API with a `bundle` that already return the same struct to the upper layer will respond like this:
```json
{
    "success": true,
    "data": {
        "success": true,
        "err": {
            "code": "",
            "msg": "",
            "ctx": null
        },
        "data": {
            "id": 1,
            "orgId": 2,
            "entityId": 0,
            "targetId": 2,
            "targetName": "erda",
            "type": "unblock-application",
            "extra": {
                "appIDs": "29",
                "end": "2021-05-12T17:50:35+08:00",
                "start": "2021-05-12T14:50:35+08:00"
            },
            "title": "erda项目，申请在时间段(2021-05-12 14:50:35~2021-05-12 17:50:35)内的部署权限",
            "priority": "",
            "desc": "aa",
            "status": "pending",
            "submitter": "2",
            "approver": "2",
            "approvalTime": "2021-05-12T17:50:56+08:00",
            "createdAt": "2021-05-12T17:50:56+08:00",
            "updatedAt": "2021-06-30T16:45:13+08:00"
        }
    },
    "err": {
        "code": "",
        "msg": "",
        "ctx": null
    }
}
```
I don't think it meets our expectations

Reply:
okay